javascript.js

```diff
@@ -0,0 +1,77 @@
+var dropdown = document.getElementById("dropdown");
+var dropdownlist = document.getElementById("dropdown__list");
+var droplink = document.getElementById("nav__link--dropdown");
+var contact = document.getElementById("nav__item--contact");
+var navbar = document.getElementById("navbar");
+const burgerBtn = document.querySelector(".navbar__hamburger");

```

Comment:
Daca asta ii `const`, ar putea fi si celelalte `const` in loc de `var`.
Folosesti `const` cand nu se schimba valoarea variabilei, gen cand selectezi elemente in cazu asta.
Alta chestie ar fi daca folosesti sintaxa de ES6, folosesti `let` si `const`, nu-ti mai trebuie `var`